src/Parser/DisciplinePriest/Modules/Features/AtonementSource.js

```diff
@@ -0,0 +1,44 @@
+import SPELLS from 'common/SPELLS';
+
+import Module from 'Parser/Core/Module';
+
+class AtonementSource extends Module {
+  _previousDamageEvent = null;
+  _previousAtonementApplicator = null;
+
+  // Spells that will apply atonement
+  atonementApplicators = new Map([
```

Comment:
This looks like a constant, so defining it above the class would be a little bit cleaner:

const ATONEMENT_APPLICATIONS = ...stuff